File: service/TFC.Application.DTO/Exercise/DeleteExercise/DeleteExerciseRequest.cs
namespace TFC.Application.DTO.Exercise.DeleteExecise
{
    public class DeleteExerciseRequest
    {
        public long? UserId { get; set; }
        public long? RoutineId { get; set; }
        public WeekDay? DayName { get; set; }
        public string? ExerciseName { get; set; }
    }
}

File: tfc_gym_app/lib/data/models/dto/exercise/delete_exercise/delete_exercise_request.dart
class DeleteExerciseRequest {
  final int routineId;
  final int dayName;
  String email;
  int exerciseId;

  DeleteExerciseRequest({
    required this.dayName,
    required this.routineId,
    required this.exerciseId,
    required this.email
  });
}